config/application.properties

```diff
@@ -1,5 +1,15 @@
 #Bridge related settings
 bridge.id=my-bridge
+
+# uncomment the following line to enable JMX Exporter metrics, check the documentation for more details
+#bridge.metrics=jmxPrometheusExporter
+
+# uncomment the following lines to enable Strimzi Reporter metrics, check the documentation for more details
+#bridge.metrics=strimziMetricsReporter
```

Comment:
From a configuration standpoint I would expect that users would just need to set `bridge.metrics=strimziMetricsReporter` and maybe the allowlist and not set the other individual configurations `kafka.prometheus.metrics.reporter.listener.enable`, `kafka.metric.reporters`.

Reply:
This is how it will work when using the Cluster Operator. Talking with @ppatierno, we agreed that, for the standalone Bridge, the user should be free to set all configurations through the properties file.

Reply:
Yeah. When bridge is running on bare metal / VM, it's the user to provide the application.properties file for the bridge with its configuration. The bridge doesn't have to change the properties file in any way enforcing other parameters which the user didn't set.